GSA/digitalgov.gov
# Removal of 1 item topics

## Summary

This PR will remove topic pages that are tagged once. 

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-removal-1-topics/)



### Solution
1. delete all references of topics within files from the front matter 
2. delete the topic from the topics folder 
3. run hugo to check that site builds 
4. added removed links to the web archive for future reference

### How To Test

1. refer to the spreadsheet, and anything that has a note that it is tagged once check that the URL is not longer active for that landing page. See Trello ticket for spreadsheet using this [Link as the base](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-removal-1-topics/)
2. also can pull down branch and run the following tests `hugo` and then `npm run lint:json`

---

### Dev Checklist
